client/src/components/Reviews/Sort.jsx

```diff
@@ -143,6 +150,10 @@ class Sort extends React.Component {
       reviewsToPassDown = this.props.newestReviews;
     }
 
+    if (this.props.filter.length > 0) {
```

Comment:
Should the if statements on lines 145 and 153 chained together?

Reply:
ooo at first glance it looks like it, but I think it's better if they are separate because they are different concepts.